website/www/site/content/en/documentation/io/built-in/google-bigquery.md

```diff
@@ -160,7 +160,9 @@ in the table.
 When writing to BigQuery, you must supply a table schema for the destination
 table that you want to write to, unless you specify a [create
 disposition](#create-disposition) of `CREATE_NEVER`. [Creating a table
-schema](#creating-a-table-schema) covers schemas in more detail.
+schema](#creating-a-table-schema) covers schemas in more detail. When updating 
```

Comment:
Should this note be included in the #creating-a-table-schema section instead?

Reply:
good call